src/api/spec/features/webui/interconnects_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'browser_helper'
+
+RSpec.feature 'Interconnects', type: :feature, js: true, vcr: true do
+  let(:admin_user) { create(:admin_user) }
+
+  scenario 'creating openSUSE.org interconnect' do
+    login admin_user
+    visit new_interconnect_path
+
+    click_button('Connect', match: :first)
+
+    visit project_show_path(project: 'openSUSE.org')
+    expect(page).to have_link('https://api.opensuse.org/public')
```

Comment:
You can also just expect the data. No need to visit the page.

Reply:
I believe the lines 10 to 13 could be replaced with:

```ruby
expect { click_button('Connect', match: :first) }.to change(RemoteProject, :count).by(1)
```

Reply:
I was not sure if this could cause flickering tests... but let's give it a try

Reply:
Between the click button and the expect on the data you can expect the flash for instance.